src/wp-includes/admin-bar.php

```diff
@@ -119,7 +119,7 @@ function wp_admin_bar_wp_menu( $wp_admin_bar ) {
 	$wp_logo_menu_args = array(
 		'id'    => 'wp-logo',
 		'title' => (
-			'<img class="cp-logo" src="' . $cp_logo_src . '" />'
+			'<img class="cp-logo" src="' . $cp_logo_src . '" alt="ClassicPress" />'
```

Comment:
better write ClassicPress Logo, if is for accessibility we need to explain what it is

Reply:
I changed it per the instructions in #171 by @nylen, see https://github.com/ClassicPress/ClassicPress/issues/171#issue-373755015 which has been open for 9 days

Reply:
I think that we need to add the logo term or someone that is blind cannot understand about what we are referencing as ClassicPress. Can be a logo, a website, the download icon etc.

Reply:
here you go